Analysis/Tutorials/src/schemaEvolution.cxx

```diff
@@ -83,9 +83,9 @@ struct CTask {
 WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
 {
   return WorkflowSpec{
-    adaptAnalysisTask<ATask>(cfgc, TaskName{"produce-etaphi-v1"}),
-    adaptAnalysisTask<BTask>(cfgc, TaskName{"produce-etaphi-v2"}),
-    adaptAnalysisTask<CTask<aod::EtaPhiV1>>(cfgc, TaskName{"consume-etaphi-v1"}), // here CTask is added with EtaPhiV1 input
-    adaptAnalysisTask<CTask<aod::EtaPhiV2>>(cfgc, TaskName{"consume-etaphi-v2"}), // here CTask is added with EtaPhiV2 input
+    adaptAnalysisTask<ProduceEtaPhiV1>(cfgc),
+    adaptAnalysisTask<ProduceEtaPhiV2>(cfgc),
+    adaptAnalysisTask<ConsumeEtaPhiVx<aod::EtaPhiV1>>(cfgc), // here ConsumeEtaPhiVx is added with EtaPhiV1 input
+    adaptAnalysisTask<ConsumeEtaPhiVx<aod::EtaPhiV2>>(cfgc), // here ConsumeEtaPhiVx is added with EtaPhiV2 input
```

Comment:
Does this run fine without TaskName in case of templates?

Reply:
Indeed, this case compiles but crashes at run time. When adding TaskName it works.